stdlib/3/_ast.pyi

```diff
@@ -11,15 +11,30 @@ class AST:
     _fields: ClassVar[typing.Tuple[str, ...]]
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     lineno: int
-    col_offset: int
+    col_offset: int  # TODO: Not all nodes have this
+    if sys.version_info >= (3, 8):
+        end_lineno: int
+        end_col_offset: int  # TODO: Not all nodes have this
+        type_comment: Optional[str]  # TODO: Only put this where it exists
 
 class mod(AST):
     ...
 
+if sys.version_info >= (3, 8):
+    class type_ignore(AST): ...
+
+    class TypeIgnore(type_ignore): ...
+
+    class FunctionType(mod):
+        argtypes = ...  # type: typing.List[expr]
+        returns = ...  # type: expr
```

Comment:
I'd prefer variable annotation syntax for new code (even if existing code in the file still uses comment syntax). Applies to classes below as well.